server/routes/api/v1/AdoptablePetsRouter.js

```diff
@@ -0,0 +1,35 @@
+import express from "express"
+import AdoptionForm from "../../../../client/components/AdoptionForm.js"
+
+import AdoptablePet from "../../../models/AdoptablePet.js"
+import AdoptionApplication from '../../../models/AdoptionApplication.js'
+

```

Comment:
There's a lot of duplication between these 2 routers, and they have the same name. Routers should also be named `camelCase`